draft-ietf-dots-signal-channel-26.xml

```diff
@@ -1034,7 +1034,7 @@
           created by the DOTS client, the DOTS server MUST return 4.00 (Bad
           Request) in the response.</t>
 
-          <t>The DOTS server couples the DOTS signal channel sessions using
+          <t>The DOTS server identifies successive DOTS signal channel sessions from the same client using
```

Comment:
IIRC we only have a single session between a given client/server at any given time, so the idea is just to notice when a new one appears and garbage-collect the old one.  I'm not fully happy with this wording, either.